tests/test_warm_reboot.py

```diff
@@ -301,13 +307,37 @@ def check_neighsyncd_timer(dvs, timer_value):
     (exitcode, num) = dvs.runcmd(['sh', '-c', "grep getWarmStartTimer /var/log/syslog | grep neighsyncd | tail -n 1 | rev | cut -d ' ' -f 1 | rev"])
     assert num.strip() == timer_value
 
+def check_redis_neigh_entries(dvs, neigh_tbl, number):
+    (exitcode, lb_output) = dvs.runcmd(['sh', '-c', "redis-cli keys NEIGH_TABLE:lo* | grep NEI | wc -l"])
```

Comment:
>redis-cli keys NEIGH_TABLE:lo* [](start = 53, length = 30)

Try prevent usage `redis-cli keys *` because if will join keys by blanks without escaping. Since you are already write python code, can you use redis-py or swsscommon ?